### PR TITLE
Feat/initial search bar

### PR DESCRIPTION
**Pull request by karljerico**
### OVERVIEW

- Fixed the logout button in the navbar

Initial Search Feature
- Add blade for the result of the search
- Created search controller that display User and Posts

### SNIPPETS

image
image
image

### CODE REVIEW

File: src/app/Http/Controllers/SearchController.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+use App\Http\Requests\SearchRequest;
+use Illuminate\Contracts\View\View;
+
+class SearchController extends Controller
+{
+    public function search(SearchRequest $request): View
+    {
+        $query = $request->input('query');
+
+        // Perform the search query on the User model
```
Comment (hipe01mark) [inline src/app/Http/Controllers/SearchController.php]: Remove comments!

Comment (hipe01mark) [reply src/app/Http/Controllers/SearchController.php]: check other file as well

File: src/app/Http/Controllers/SearchController.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        $userResults = User::where('name', 'like', "%$query%")->get();
+
+        // Perform the search query on the Post model
+        $postResults = Post::where('content', 'like', "%$query%")->get();
+
+        return view('search.results', compact('userResults', 'postResults'));
```
Comment (hipe01mark) [inline src/app/Http/Controllers/SearchController.php]: Why this is being exposed in the controller? You don't have service layer for this??

**Review by hipe-francis — DISMISSED** [review]
LGTM 🚀

File: src/app/Http/Controllers/SearchController.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+class SearchController extends Controller
+{
+    private $searchService;
+
+    public function __construct(SearchService $searchService)
+    {
+        $this->searchService = $searchService;
+    }
+
```
Comment (hipe01mark) [inline src/app/Http/Controllers/SearchController.php:20]: Add comment!

**Review by hipe01mark — DISMISSED** [review]
LGTM 🙇

**Review by hipe-francis — APPROVED** [review]
LGTM 🚀

**Review by hipe01mark — APPROVED** [review]
LGTM 🙇

**Pull request merged**